code/modules/mob/living/simple_animal/hostile/russian.dm

```diff
@@ -26,27 +26,33 @@
 	unsuitable_atmos_damage = 7.5
 	faction = list("russian")
 	status_flags = CANPUSH
-	del_on_death = 1
-
 	footstep_type = FOOTSTEP_MOB_SHOE
+	del_on_death = TRUE
+	var/mob_spawner = /obj/effect/mob_spawn/corpse/human/russian
+	var/held_item = /obj/item/knife/kitchen
```

Comment:
Autodoc

Reply:
I'd ask you to give all of these one parent type with this shit but simple animals arent long for this world so w/e